### DISCUSSION TITLE
`qsv searchset` regex argument clarification

### DISCUSSION THREAD

**Discussion by rzmk** [dathere/qsv#2156] [Q&A] [answered]
For `qsv searchset`, wouldn't the `<regex>` argument be `<regexset-file>`?

https://github.com/jqnatividad/qsv/blob/5202508e5c3969b279c20cf80bb1e37d89afd826/src/cmd/searchset.rs#L29-L31

**Comment by jqnatividad** [comment]
You're right @rzmk ! This is a lazy cut n' paste bug...

**Reply by jqnatividad** [reply, marked as answer]
Fixed by https://github.com/jqnatividad/qsv/issues/2161
